cli/lib/kontena/cli/vault/import_command.rb

```diff
@@ -0,0 +1,58 @@
+module Kontena::Cli::Vault
+  class ImportCommand < Kontena::Command
+    include Kontena::Cli::Common
+    include Kontena::Cli::GridOptions
+
+    option ['-u', '--upsert'], :flag, 'Create secret unless already exists', default: false
+    parameter "FILENAME", "Secret yaml file"
+
+    def execute
+      require_api_url
+      require_current_grid
+      token = require_token
+
+      # Not sure of a better way to do this for now. Skips the first two ARGF
+      # reads in order to get to the third without raising an error.
+      2.times do
```

Comment:
This is the piece I really don't like. If anyone has any thoughts on how to improve it, I'm all ears.